### PR TITLE
TDH - Prime Restore UI change

### PR DESCRIPTION
**Pull request by pnaw94**
@fyniac @rowenaluk 
scr01
scr02

### CODE REVIEW

**Comment by fyniac** [comment]
@snopoke can you review?

**Comment by snopoke** [comment]
This seems to remove the existing page completely. The spec says:

> 1. Make the current page a hidden ‘advanced’ page: Take the existing UI at https://www.commcarehq.org/a/rec/phone/prime_restore/. Move it to https://www.commcarehq.org/a/rec/phone/prime_restore/advanced. You do NOT need to link to this from anywhere else on CommCareHQ. 
2. Simplify the current page.  Take a copy of the existing UI and make the following changes:

i.e. the current page should still be accessible with all the options and there should be a new page with just the one button on it.

**Comment by snopoke** [comment]
@pnaw94 See last comment
cc @fyniac

**Comment by pnaw94** [comment]
@snopoke updated

File: corehq/apps/style/templates/style/includes/file_download.html [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
     <script src="{% static 'jquery/dist/jquery.min.js' %}"></script>
     <script src="{% static 'hqstyle/js/jquery/jquery-migrate-1.2.1.min.js' %}"></script>
+    <script src="{% new_static 'bootstrap/dist/js/bootstrap.js' %}"></script>
     {{ block.super }}
 {% endblock %}
+
+{% block content %}
+    <div class="col-sm-12">
+        {{ block.super }}
+        <a href="{% url 'my_project_settings' domain %}" class="btn btn-primary">{% trans 'Return to Project Settings' %}</a>
```
Comment (snopoke) [inline corehq/apps/style/templates/style/includes/file_download.html]: should `domain` be `{{domain}}`

Comment (snopoke) [inline corehq/apps/style/templates/style/includes/file_download.html]: Also is this compatible with the other usages that don't supply a domain? e.g. data exports?

**Comment by snopoke** [comment]
looks good, just the question about the soil download view

**Comment by pnaw94** [comment]
@snopoke 
The template is actually used only in the view below, so I think it is safe to leave template like it is.
```
@login_and_domain_required
def retrieve_download(req, domain, download_id, template="style/includes/file_download.html")
```

**Pull request merged**